src/main/java/org/gridsuite/securityanalysis/server/dto/SecurityAnalysisParametersValues.java

```diff
@@ -0,0 +1,36 @@
+/**
+ Copyright (c) 2023, RTE (http://www.rte-france.com)
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.securityanalysis.server.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.gridsuite.securityanalysis.server.entities.SecurityAnalysisParametersEntity;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+@Getter
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
+public class SecurityAnalysisParametersValues {
```

Comment:
```suggestion
public class SecurityAnalysisParametersInfos {
```
Usually we use this suffix no ? I know it was called like that in study-server but it's the occasion to fix it

Reply:
SecurityAnalysisParametersInfos already exists that's why

Reply:
Ah OK I see, my bad